pr_agent/algo/pr_processing.py

```diff
@@ -78,9 +78,9 @@ def get_pr_diff(git_provider: GitProvider, token_handler: TokenHandler, model: s
     return final_diff
 
 
-def pr_generate_extended_diff(pr_languages: list, token_handler: TokenHandler,
-                              add_line_numbers_to_hunks: bool) -> \
-        Tuple[list, int]:
+def pr_generate_extended_diff(pr_languages: list,
+                              token_handler: TokenHandler,
+                              add_line_numbers_to_hunks: bool) -> Tuple[list, int, list]:
     """
     Generate a standard diff string with patch extension, while counting the number of tokens used and applying diff
     minimization techniques if needed.
```

Comment:
**Suggestion:** The docstring for the `pr_generate_extended_diff` function should be updated to include the new return value `patches_extended_tokens`.
```suggestion
def pr_generate_extended_diff(pr_languages: list,
                              token_handler: TokenHandler,
                              add_line_numbers_to_hunks: bool) -> Tuple[list, int, list]:
    """
    Generate a standard diff string with patch extension, while counting the number of tokens used and applying diff
    minimization techniques if needed.
    Returns a list of extended patches, the total number of tokens used, and a list of tokens for each extended patch.
```